src/main/grammars/RustParser.bnf

```diff
@@ -1102,6 +1098,9 @@ item_macro_arg ::= '(' token_trees ')' ';'
 try_macro_args ::= <<macro_args_delim any_expr>>
 vec_macro_args ::= '[' expr_list ']' | '[' any_expr';' any_expr ']' | '[' ']'  { pin = 1 }
 
+private zz_log_macro_args ::= ('target' ':' expr ','?)? (<<comma_separated_list format_macro_arg>>)?
```

Comment:
From the definition of trace https://doc.rust-lang.org/log/log/macro.log.html this is normal usage.
But I can add `token trees` fallback as well.